MediaWikiKit/MediaWikiKit/MWKSection.m

```diff
@@ -106,6 +106,10 @@ - (MWKTitle*)sourceTitle {
     }
 }
 
+- (BOOL)hasTextData {
+    return [self.article.dataStore htmlFileExistsForSectionId:self.sectionId title:self.article.title];
+}
```

Comment:
@montehurd thanks for the heads-up (for posterity, even though we talked about it over hangout), but this info should generally be evident in comments and/or tests.  The idea situation is for the code or tests to be as expressive and self-documenting as possible.